src/applications/find-va-forms/components/SearchResult.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import moment from 'moment';
+
+import * as customPropTypes from '../prop-types';
+
+export default function SearchResult({ form }) {
+  const pdf = form.attributes.url.toLowerCase().includes('.pdf') ? '(PDF)' : '';
```

Comment:
Should we protect for null/undefined values here with something like `form?.attributes?.url`? Same with `form?.attributes?.lastRevisionOn` below 🙂

Reply:
I'd like to think this value will never be null since it's a required part of the UX and the API enforces that it exists, buuut it would also be bad if a runtime error on a single form crashed the whole widget if an empty form did slip through. I'll add a safety check in there.